scripts/hdiff.sh

```diff
@@ -0,0 +1,8 @@
+base=$(basename $1 .test)
+
+out=$(find . -name ${base}.out)
+exp=$(find . -name ${base}.regexpect)
+
+ORION_DIFF=${ORION_DIFF:-tkdiff}
+
+$ORION_DIFF $out $exp
```

Comment:
Yeah, I was thinking about that and the reason NOT to do it is that, I NEVER set CB_DIFF_TOOL in any startup script, if I want graphical diffs in my functest, I give it to testHarness.sh like this:

    CB_DIFF_TOOL=tkdiff testHarness.sh

hdiff.sh is different, it needs an env var to be set, and it would be set in the startup files, of course.

That's why I chose another env var